tests/unit/test_charm_2.py

```diff
@@ -393,3 +393,43 @@ def test_core_reconciliation(self):
             self.installed_database_hosts,
             "wordpress should be installed after database config changed"
         )
+
+    @staticmethod
+    def _gen_action_event_mock():
+        event_mock = unittest.mock.MagicMock()
+        event_mock.set_results = unittest.mock.MagicMock()
+        event_mock.fail = unittest.mock.MagicMock()
+        return event_mock
+
+    def test_get_initial_password_action_before_replica_consensus(self):
+        """
+        arrange: before peer relation established but after charm created
+        act: run get-initial-password action
+        assert: get-initial-password action should fail
+        """
+        self.harness.begin_with_initial_hooks()
```

Comment:
Is it worth separating the act and assert here? Similar comment applies to the other tests

Reply:
Fixed, thanks!